reference/forms/types/options/label_format.rst.inc

```diff
@@ -0,0 +1,38 @@
+label_format
+~~~~~~~~~~~~
+
+.. versionadded:: 2.6
+    The ``label_format`` option was introduced in Symfony 2.6.
+
+**type**: ``string`` **default**: ``null``
+
+Sets the format used to generate the ``<label>`` element. This is usefull when
```

Comment:
I have some concerns with the first phrase. Since the option is called `label_format`, the explanation looks a bit obvious and "empty": `Sets the format used to generate the ``<label>`` element.` What does "format" exactly mean and what does it generate: the entire `<label>` tag, just the label name, etc.